docs/installing.md

```diff
@@ -1,25 +1,50 @@
-# Running a local Haveno test network
+# Set up environment
 
-These are the steps needed to set up Haveno test instances.
+These are the steps needed to build Haveno and test it on our test network or locally.
 
-## 1. Install dependencies
+## Install dependencies
 
-On Ubuntu: `sudo apt install make wget git git-lfs openjdk-11-jdk`. The Bitcoin and Monero binaries will be downloaded and verified automatically in the next step.
+On Ubuntu: `sudo apt install make wget git git-lfs openjdk-11-jdk`. The Monero binaries will be downloaded and verified automatically in the next step.
 
-## 2. Build Haveno
+## Build Haveno
 
 1. Download this repository: `git clone https://github.com/haveno-dex/haveno.git`
-2. Navigate to the root of the repository (`cd haveno`) and build the repository: run `make` in the terminal and wait until the process is completed (this will also download and verify the Monero and Bitcoin binaries).
+2. Navigate to the root of the repository (`cd haveno`) and build the repository: run `make` in the terminal and wait until the process is completed (this will also download and verify the Monero binaries).
 
-## 3. Run a local XMR testnet
+## Join the public testnet
+
+If you want to try Haveno in a live setup, Launch an Haveno instance that will connect to other peers on our public test environment, which runs on Monero's stagenet. You'll be able to make test trades with other users and have a preview of Haveno's trade protocol in action. Note that development is very much ongoing. Things are slow and might break.
+
+Steps:
+
+1. Run `make testuser-desktop`
+2. A monero address will be printed in the terminal, the string will look like this:
+
+```
+Monero wallet primary address: 58GBfHgSvji6SU9gV4deERTRWsx2hFgXBDEFpSXdwhyADCU6VB7Nexb7JNzFyiBdKhX71aCFtyhN9fXBA9F8VtGRGZpSrYe
+```
+
+Copy this address.
+
+3. Go to the [stagenet faucet](https://community.rino.io/faucet/stagenet/) and past the address above in the "Get XMR" field. Submit and see the stagenet coins being sent to your Haveno instance.
+4. While you wait the 10 confirmations (20 minutes) needed for your funds to be spendable, create a fiat account by clicking on "Account" in the top menu, select the "National currency accounts" tab, then add a new account. For simplicity, we suggest to test using a Revolut account with a random ID.
+5. Now pick up an existing trade or open a new one. Fund your trade and wait 10 blocks for your deposit to be confirmed.
```

Comment:
for your deposit to be *unlocked?

The deposit is technically confirmed after one block, so we should avoid misusing this term.